lib/tasks/unpublish_ukri_content.rake

```diff
@@ -0,0 +1,195 @@
+    { govuk_page: "https://www.gov.uk/government/publications/clean-and-affordable-energy-projects-address-the-challenge" },
+    { govuk_page: "https://www.gov.uk/government/publications/collaboration-nation-digital-services-2010" },
+    { govuk_page: "https://www.gov.uk/government/publications/collaboration-nation-high-value-manufacturing-2013" },
+    { govuk_page: "https://www.gov.uk/government/publications/future-cities-uk-creating-better-places-to-live-work-and-play" },
+    { govuk_page: "https://www.gov.uk/government/publications/icure-evaluation-of-pilot-programme" },
+    { govuk_page: "https://www.gov.uk/government/publications/innovate-uk-10-years-shaping-the-future" },
+    { govuk_page: "https://www.gov.uk/government/publications/innovate-uk-aerospace-sme-case-studies" },
+    { govuk_page: "https://www.gov.uk/government/publications/innovation-with-impact-50-companies-succeeding-with-innovate-uk" },
+    { govuk_page: "https://www.gov.uk/government/publications/knowledge-transfer-partnerships-achievements-and-outcomes-2013-to-2014" },
+    { govuk_page: "https://www.gov.uk/government/publications/london-and-cambridge-internet-of-things-directory-of-projects" },
+    { govuk_page: "https://www.gov.uk/government/publications/low-and-zero-emission-vehicles-impact-review-2018" },
+    { govuk_page: "https://www.gov.uk/government/publications/smart-funding-assessment-of-impact-and-evaluation-of-processes" },
+    { govuk_page: "https://www.gov.uk/government/publications/the-economic-impact-on-the-uk-of-a-disruption-to-gnss" },
+    { govuk_page: "https://www.gov.uk/government/publications/the-knowledge-transfer-partnership-programme-an-impact-review" },
+    { govuk_page: "https://www.gov.uk/government/collections/horizon-2020-the-eu-research-and-innovation-funding-programme" },
+    { govuk_page: "https://www.gov.uk/government/case-studies/power-roll-bringing-cheap-solar-power-to-africa-and-india" },
+    { govuk_page: "https://www.gov.uk/government/publications/the-low-carbon-vehicles-innovation-platform-an-impact-review" },
+  ]
+
+  content_to_unpublish.each do |url|
+    slug = URI.parse(url[:govuk_page]).path.split("/").last
+
+    # Rebecca P's user id
+    user = User.find(9929)
+
+    published_edition = Document.find_by(slug: slug)&.published_edition
+
+    unless published_edition
+      puts "No published edition for GOV.UK page: #{url[:govuk_page]}"
+      next
+    end
+
+    if (pre_pub_edition = published_edition.other_editions.in_pre_publication_state.first)
+      edition_deleter = Whitehall.edition_services.deleter(pre_pub_edition)
+
+      if edition_deleter.perform!
+        puts "Draft deleted with URL: #{url[:govuk_page]}"
+      else
+        puts edition_deleter.failure_reason
+      end
+    end
+
+    unpublishing_params = {
+      redirect: false,
+      unpublishing_reason_id: UnpublishingReason::Withdrawn.id,
+      explanation: "withdrawn with content moved to ukri",
```

Comment:
Does this message appear in a banner on the public site?  If it does, could you clarify with UKRI what content they would like displayed.

Reply:
Yes it does, I'll send them a message :)